test-files/cases/hoot-js/geometry-generic-1/README.txt

```diff
@@ -0,0 +1,51 @@
+This tests generic geometry conflation, which is conflation meant to occur against features that are not conflatable by any specific conflation
```

Comment:
This describes the main case test for generic geometry conflation. The main idea here is to make sure generic geometries get conflated by the new scripts and that nothing that should be conflated by specific conflation routines gets conflated by them.